Makefile

```diff
@@ -0,0 +1,211 @@
+# Orchestrator
+ORCHESTRATOR_PATH := $(shell pwd)
+
+# Bootstrapper
+OPERATOR_ADDRESS := 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
+BOOTSTRAP_OUTPUT_PATH := $(shell pwd)/build/bootstrap.json
+BOOTSTRAPPER_COMMIT := b0b647500c2ae3e3b0d99e345fa652989bca4726
+BOOTSTRAPPER_PATH := $(shell pwd)/madara-bootstrapper
+VERIFIER_ADDRESS := 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
+
+# SNOS
+CAIRO_LANG_COMMIT := a86e92bfde9c171c0856d7b46580c66e004922f3
+
+# Madara
+MADARA_PATH := $(shell pwd)/madara
+MADARA_COMMIT := 93d57632e7a56707d74ab63eb0ca8558df9f0d8e
+MADARA_DATA_PATH := $(shell pwd)/build/madara
+
+# Pathfinder
+PATHFINDER_COMMIT := 138140e5fd967ede92806fd62de47c2e6b65712a
+PATHFINDER_DATA_PATH := $(shell pwd)/build/pathfinder
+PATHFINDER_PATH := $(shell pwd)/pathfinder
+
+# Environment file
+ENV_FILE := $(shell pwd)/.makefile.json
+PID_FILE := $(shell pwd)/.pids.json
+
+.PHONY: all anvil madara core-contract update-madara eth-bridge pathfinder orchestrator cleanup
+
+# JSON update functions
+
+define save_pid
+	./scripts/save_pid.sh $1 $2
+endef
+
+define get_pid
+	./scripts/get_pid.sh $1
+endef
+
+define kill_pid
+	./scripts/kill_pid.sh $1
+endef
+
+define save_json
+	echo "saving json\n"
+	sh ./scripts/save_json.sh $1 $2
+endef
+
+setup:
+	@echo "Starting setup process..."
+	@make cleanup
+	@trap 'make cleanup' EXIT INT TERM
+
+	# @echo "Starting Anvil..."
+	# @make anvil & $(call save_pid,anvil,$$!)
```

Comment:
Seems like anvil is being used down below at L109, any reason to have this comment here, if not let's please remove it.

Reply:
good catch. so i actually planned to remove from here itself because later, maybe you want to use sepolia to set it up. so if you want anvil you set it up separately. although there are other changes you might need to do as we've never tested makefile with sepolia, but that's the mental model.